ci-operator/jobs/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master-presubmits.yaml

```diff
@@ -1,5 +1,118 @@
 presubmits:
   openshift/cluster-ingress-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e-aws
+    decorate: true
+    name: pull-ci-openshift-cluster-ingress-operator-master-e2e-aws
+    rerun_command: /test e2e-aws
+    skip_cloning: true
+    spec:
+      containers:
+      - command:
+        - ci-operator
+        - --give-pr-author-access-to-namespace=true
+        - --artifact-dir=$(ARTIFACTS)
+        - --secret-dir=/usr/local/e2e-aws-cluster-profile
+        - --template=/usr/local/e2e-aws
+        - --target=e2e-aws
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: master.yaml
+              name: ci-operator-openshift-cluster-ingress-operator
+        - name: JOB_NAME_SAFE
+          value: e2e-aws
+        - name: TEST_FOCUS
```

Comment:
Why just one test?

Reply:
It's a smoke test, as the installer team is able to reliably get the existing suite to pass, we'll add more.

Right now, just verifying a cluster came up is the key objective.  The suite value would change across all repos, not the job definition.